### ISSUE TITLE
PoB window : all black/void

### ISSUE THREAD

**Issue by Korbadi** [Openarl/PathOfBuilding#647]
When I open PoB, the window open like usual but the content is just black, no tree or anything. There's still the upper bar and I can click on it (like on the "close tab button" but it takes several seconds to take effect). 
Sometimes it opens normally, I can see the tree and do stuff and then it goes black...

I tried reinstalling PoB 3-4 times with the same result.

PoB Release : 1.4.59
Pc : windows 10

*I'll link visuals later*

**Comment by Openarl** [comment]
Visuals would be nice. I would suggest updating your graphics driver if you haven't already done so, as it often fixes issues like this.

**Comment by krab** [comment]
Yeah program starts like this: https://i.imgur.com/pCIL1iX.png and loads fine, but when you change window size - close - start again: https://i.imgur.com/hMKO0Mz.jpg

**Comment by Dasky14** [comment]
Still having almost the same issue, check #672 for more info. @Openarl
